geomesa-spark/geomesa-spark-core/src/main/scala/org/locationtech/geomesa/spark/package.scala

```diff
@@ -0,0 +1,37 @@
+/***********************************************************************
+ * Copyright (c) 2013-2019 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ ***********************************************************************/
+
+package org.locationtech.geomesa
+
+import org.opengis.feature.simple.SimpleFeatureType
+
+package object spark {
+
+  trait Schema {
+    def schema: SimpleFeatureType
+  }
+
+  // Resolve issue with wrapped instance of org.apache.spark.sql.execution.datasources.CaseInsensitiveMap in Scala 2.10
+  object CaseInsensitiveMapFix {
```

Comment:
Since we don't use 2.10 anymore, can this go away?

Reply:
I wasn't sure exactly what this was fixing, so I figured I'd leave it... I can remove it and see if anything breaks.

Reply:
Yeah, let's try and remove it at some point.  Do you want to do that now or enter a separate ticket?

Reply:
I'll remove it here, thanks

Reply:
removed